src/schema/contentful/phoenix.js

```diff
@@ -310,13 +378,18 @@ const resolvers = {
       Loader(context, preview).campaignWebsites.load(id),
     campaignWebsiteByCampaignId: (_, { campaignId, preview }, context) =>
       Loader(context, preview).campaignWebsiteByCampaignIds.load(campaignId),
+    pages: (_, { id, preview }, context) =>
```

Comment:
This one should be singular
```suggestion
    page: (_, { id, preview }, context) =>
```